frontend/packages/console-plugin-sdk/src/typings/clusterserviceversions.ts

```diff
@@ -11,6 +11,8 @@ namespace ExtensionProperties {
     apiGroup: string;
     /** action callback */
     callback: (kind: K8sResourceKindReference, obj: any) => () => any;
+    required?: string | string[];
```

Comment:
Do we really need to support two different types for this? Can it just be `string[]`, and if it's a single value, the array will just be length 1.

Reply:
https://github.com/openshift/console/blob/master/frontend/packages/console-plugin-sdk/src/registry.ts#L175-L178
Keeping it consistent with this.